diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -47,6 +50,122 @@ private fun generateWarningNames() {
     kotlinFile.writeTo(File("diktat-rules/src/main/kotlin"))  // fixme: need to add it to pom
 }
 
+private fun generateCodeStyle() {
+    val file = File("info/guide/diktat-coding-convention.md")
+    val appendixFile = File("wp/sections/appendix.tex").readLines().toMutableList()
```

Comment:
```suggestion
    val appendixFileLines = File("wp/sections/appendix.tex").readLines().toMutableList()
```
I forgot that it's a list and not a File by the point where it is actually used.